### ISSUE TITLE
Training setting different from the original paper

### ISSUE THREAD

**Issue by ZENGXH** [Yugnaynehc/banet#3]
Hi, thanks for your great work! 

I noticed that some setting in your implementation is different from the original paper, such as the learning rate, batch_size, weight initialization and the optimizer. I am wondering if there is any reason behind that? Have you tried the original setting before? 

Best

**Comment by Yugnaynehc** [comment]
Hi, sorry for replying late! Yes, there are many difference between this implementation and original paper. The original settings seem not work well in this implementation, so I make some modification to them.
